docs/getting-started/smr-disk.md

```diff
@@ -250,14 +250,16 @@ also be checked using the [*libzbc* conformance test suite](/tests/zbc-tests).
 
 ### Verifying The Disk
 
-Assuming that a compatible HBA is being used, after connecting the drive and
-eventually rebooting the system (most SAS HBAs have plug-and-play features in
-which case rebooting the system after connecting or disconnecting a disk is
-not necessary), verifying the disk identification and checking the disk
-parameters and zone configuration can be done in the exact same manner as with
-Serial ATA disks as discussed above.
-
-In these examples, `/dev/sdc` is a SAS disk connected to a SAS HBA and
+If you are using a compatible HBA and you have connected the drive and rebooted
+the system, verifying the disk identification and checking the disk parameters
+and zone configuration can be done in the exact same manner as with Serial ATA
+disks as discussed above.
+
+(Remember that most SAS HBAs have plug-and-play features that make it
+unnecessary to reboot the system after connecting or disconnecting a disk to
+it.) 
```

Comment:
May be make this a note box ? E.g.

!!! Note
      Most SAS HBAs have plug-and-play features that make it unnecessary to reboot the
      system after connecting or disconnecting a disk to it.

Reply:
I've done this, but I can't seem to get it to render the way other notes (and indeed, other notes that I myself have written) render. Let me know if you have any ideas about how this can be fixed.

Reply:
Update: Turns out you need four spaces at the beginning of all lines in the note in order for the text in the note to render correctly.

What madness.